api-docs-slate/source/includes/_clients.md

```diff
@@ -17,94 +17,99 @@ testnet   | 18332
 regtest   | 48332
 simnet    | 18556
 
-You can interact with bcoin with REST Api as well as RPC,
-there are couple of ways you can use API.
+You can interact with bcoin with its REST API as well as with RPC.
+There are couple of ways you can use the API:
 
-- `bcoin-cli` - has almost all methods described to be used.
-- `javascript` - Clients used by `bcoin-cli` can be used directly from javascript
-- `curl` - or you can use direct HTTP calls for invoking REST/RPC API calls.
+- `bcoin-cli` - methods built specifically into bcoin by its developers
+- `bcoin-cli rpc` - adds functionality that mimics Bitcoin Core RPC
+- `javascript` - methods used by `bcoin-cli` can be accessed directly from javascript
+- `curl` - you can use direct HTTP calls for invoking both REST and RPC API calls
 
-## Configuring BCOIN CLI
+## Configuring bcoin-cli
 
 ```shell--visible
-# You can use config file
-bcoin-cli --config /full/path/to/bcoin.conf
-
-# Or with prefix (which will later load bcoin.conf file from the directory)
-bcoin-cli --prefix /full/path/to/bcoin/dir
-
 # You can configure it by passing arguments:
 bcoin-cli --network=regtest info
 bcoin-cli info --network=regtest
 
-# Or use ENV variables (Starting with BCOIN_)
+# Or use environment variables (Starting with BCOIN_)
 export BCOIN_NETWORK=regtest
-export BCOIN_API_KEY=yoursecret
+export BCOIN_API_KEY=api-key
```

Comment:
maybe preface with a `$` to denote a variable and show that this is just a place holder